tools/buildsys/src/manifest.rs

```diff
@@ -167,6 +174,13 @@ pub(crate) enum ImageFormat {
     Vmdk,
 }
 
+#[derive(Deserialize, Debug, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub(crate) enum SupportedArch {
+    X86_64,
```

Comment:
That's a suspicious variant name wrt Rust style. Not sure what to suggest though.

Reply:
Amd64 and Arm64 are always options, but they are annoyingly similar if you aren't paying attention.

Reply:
> That's a suspicious variant name wrt Rust style. Not sure what to suggest though.

I think we can use lowercase names and tell rust/clippy to ignore the warning?

> Amd64 and Arm64 are always options

We may want to use serde aliases to support these alternate names without having extra variants.

Reply:
I feel like the names are fine, I'd rather not ignore clippy warnings just for a lowercase letter.

I'm leaning towards not adding the aliases.  Our Makefile (which calls `buildsys`) only allows `x86_64` and `aarch64`.  In order to even use an alias, you'd need to be calling `buildsys` directly which is probably pretty unlikely.